ld.implicit.script

```diff
@@ -0,0 +1,8 @@
+  SECTIONS {
```

Comment:
Could you move this to `/src/`?

Reply:
@refack Any particular reason do you feel to create new directory? It is new license or something else?

Reply:
In general I'm in favor of more fine grained structuring for the tree. This seems like a cohesive module, so it seems like a good candidate.
And yes, that it is under a different license makes it unique. Might even need to be a `/dep/`.
Also having a readme in that directory will help document this for posterity.

Reply:
+1 to change, haven't noticed it was in the root of the repo.
I'd say at least to simply group the files in one place, this will make maintenance and understanding of the implementation easier IMO.

Reply:
> Might even need to be a `/dep/`.

Please don’t move files to `deps/` that are not imported and updated from an upstream repo. 

> And yes, that it is under a different license makes it unique.

It is not under a different license, and therefore not unique in that respect.

Reply:
> It is not under a different license, and therefore not unique in that respect.

Yes it is, see https://github.com/nodejs/node/pull/22079#discussion_r207097339
Neither of us are layers, but I'd rather be safe than sorry.

Reply:
As simple solution might be to put it in `/src/large_pages/` and exclude that directory in our LICENSE.

Reply:
Moving it to a different directory and containing it sounds good to me. @uttampawar and I will work on it.

Reply:
Thank you 🎉

Reply:
@refack Addressed in the latest updated PR. Can you explain your thoughts about LICENSE file content, please?